test/widgets/channel_colors_test.dart

```diff
@@ -137,12 +137,12 @@ void main() {
 
       test('iconOnPlainBackground', () {
         void runCheck(int base, Color expected) {
-          check(StreamColorSwatch.light(base)).iconOnPlainBackground.equals(expected);
+          check(ChannelColorSwatch.light(base)).iconOnPlainBackground.equals(expected);
         }
 
         // Check against everything in ZULIP_ASSIGNMENT_COLORS
         // in <https://replit.com/@VladKorobov/zulip-topic-feed-colors#script.js>.
-        // (Skipping `streamColors` because there are 100+ of them.)
+        // (Skipping `channelColors` because there are 100+ of them.)
```

Comment:
This is referring to an identifier, so should be updated at the same time the identifier is.

In this case the identifier is in the linked piece of code on replit.com, which hasn't been updated (and presumably won't be, as it's a historical piece of showing-one's-work). So this should continue to refer to the name that actually appears there.